### PR TITLE
AWS v18.2.0

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/roadmap/issues/1860

### Checklist
- [x] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### CODE REVIEW

File: aws/v18.2.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Generated with:                                                # Generated with:
+# devctl release create --name v18.1.0 --provider aws --base v | # devctl release create --name v18.2.0 --provider aws --base v
+apiVersion: release.giantswarm.io/v1alpha1                       apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                    kind: Release
+metadata:                                                        metadata:
+  annotations:                                                     annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/c       giantswarm.io/docs: https://docs.giantswarm.io/reference/c
+  creationTimestamp: null                                          creationTimestamp: null
+  name: v18.1.0                                                |   name: v18.2.0
```
Comment (T-Kukawka) [inline aws/v18.2.0/release.diff:9]: why 18.1.0 not 18.1.1?

Comment (njuettner) [reply aws/v18.2.0/release.diff]: My bad, the only thing what changed in v18.1.1 was aws-operator. But I fixed it 👍🏻

**Comment by njuettner** [comment]
I can see the current test pipeline is currently broken. I'm trying to test this manually for now.

**Comment by njuettner** [comment]
Waiting for a decision on https://github.com/giantswarm/aws-operator/pull/3477

**Comment by njuettner** [comment]
@T-Kukawka PTAL 🙏🏻, only announcement need to be prepared

**Comment by LolloneS** [comment]
What about:
- fixed API server monitoring
- new node-operator version
?

I don't see them in the changelog, are they in this release? If so let's highlight them

File: aws/v18.2.0/README.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
+# :zap: Giant Swarm Release v18.2.0 for AWS :zap:
+
+This release contains changes that address several Kubernetes vulnerabilities bringing it to the latest `1.23.15` version. 
+
+Additionally to allow customers better cost optimatization the size of [EBS Volumes for Logging, Docker and Containerd can be now modified](https://docs.giantswarm.io/use-the-api/management-api/crd/awsmachinedeployments.infrastructure.giantswarm.io/#v1alpha2). After analysis of the Logging EBS Volume usage itself, the default has been changed from 100GB to 15GB per node for cost savings. Please contact your Account Engineer to start the analysis of the Docker and Contanerd Volumes sizes for the targets to be modified.
```
Comment (njuettner) [inline aws/v18.2.0/README.md:5]:
```suggestion
Additionally to allow customers better cost optimatization the size of [EBS Volumes for Logging, Docker and Containerd can be now modified](https://docs.giantswarm.io/use-the-api/management-api/crd/awsmachinedeployments.infrastructure.giantswarm.io/#v1alpha3). After analysis of the Logging EBS Volume usage itself, the default has been changed from 100GB to 15GB per node for cost savings. Please contact your Account Engineer to start the analysis of the Docker and Contanerd Volumes sizes for the targets to be modified.
```

**Comment by T-Kukawka** [comment]
> What about:
> 
>     * fixed API server monitoring
> 
>     * new node-operator version
>       ?
> 
> 
> I don't see them in the changelog, are they in this release? If so let's highlight them

Node-exporter is not tied to the release, the monitoring fixes have been now included

**Review by LolloneS — APPROVED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**